ENG-2185: home page redesign and quickstarts landing page (#78)
Basis-Theory/basistheory-integration-guides: index.markdown

        <meta charset="utf-8">
        <title>Basis Theory Overview</title>
    </head>
    <body class="home-page">
        <h1>Documentation</h1>
        <h5>Explore our APIs, SDKs, and all of the tools you need to make your sensitive data more secure and usable.</h5>
        <div class="quickstarts">
            <div class="quickstart-info">
                <div class="quickstart-header">Quickstarts</div>

>                 <div class="quickstart-subtitle">Learn how to use Basis Theory use tokenization to encrypt personal information and safely share it to Auth0.</div>

Reviewer: ```suggestion
                <div class="quickstart-subtitle">Get a foundational understanding of how to secure your data with Basis Theory allowing you to no longer worry about the underlying storage or encryption of the data.</div>
```

            </div>
        </div>
        <div class="sub-header">
            Concepts
        </div>
        <div class="cards">
            <a class="card" href="/concepts/what-are-tokens">
                <div class="icon-and-time-estimate">
                    <img src="./assets/images/icons/blue-token.svg">
                    <div class="time-estimate">
                        <img src="./assets/images/icons/grey-time.svg" alt="time-icon">
                        <div>5 min</div>
                    </div>
                </div>
                <div class="card-title">What are Tokens?</div>
                <div>Use Tokens to secure and share your sensitive data.</div>
            </a>
            <a class="card" href="/concepts/what-are-tenants">
                <div class="icon-and-time-estimate">
                    <img src="./assets/images/icons/blue-terminal.svg">
                    <div class="time-estimate">
                        <img src="./assets/images/icons/grey-time.svg" alt="time-icon">
                        <div>5 min</div>
                    </div>
                </div>
                <div class="card-title">What are Tenants?</div>

>                 <div>Where your Tokens and Applications are defined, managed, and stored.</div>

Reviewer: ```suggestion
                <div>Tenats are where Tokens and Applications are defined, managed, and stored.</div>
```

            </a>
            <a class="card" href="/concepts/what-is-the-proxy">
                <div class="icon-and-time-estimate">
                    <img src="./assets/images/icons/blue-reactor.svg">
                    <div class="time-estimate">
                        <img src="./assets/images/icons/grey-time.svg" alt="time-icon">
                        <div>5 min</div>
                    </div>
                </div>
                <div class="card-title">What is the Proxy?</div>

>                 <div>Send tokenized data without touching it.</div>

Reviewer: ```suggestion
                <div>Proxy enables making HTTP requests containing sensitive tokenized data to your 3rd party partners, without your systems needing access to the raw data.</div>
```
